mapie/metrics.py

```diff
@@ -1575,3 +1575,60 @@ def spiegelhalter_p_value(y_true: NDArray, y_score: NDArray) -> float:
     sp_stat = spiegelhalter_statistic(y_true, y_score)
     sp_p_value = 1 - scipy.stats.norm.cdf(sp_stat)
     return sp_p_value
+
+
+def regression_MWI_score(
+        y_true: NDArray,
+        y_pis: NDArray,
+        alpha: float
+) -> float:
+    """
+    This is an implementation of the Winkler interval score
+    (https://otexts.com/fpp3/distaccuracy.html#winkler-score).
+
```

Comment:
Could you add a short description of the score or the formula rather than a link ?